app/schema/answer.py

```diff
@@ -100,3 +100,14 @@ def mandatory_error(self, state):
         state.errors = []
         state.errors.append(self.questionnaire.get_error_message('MANDATORY', self.id))
         state.is_valid = False
+
+    @staticmethod
+    def _valid_option_selected(state_input, options):
```

Comment:
Put it here after sitting in a meeting that said other option value will always be visible, this brings radio boxes into play and they will also need this